--- a/sapi/apache2handler/config.m4
+++ b/sapi/apache2handler/config.m4
@@ -64,18 +64,10 @@ if test "$PHP_APXS2" != "no"; then
   fi
 
   APXS_LIBEXECDIR='$(INSTALL_ROOT)'`$APXS -q LIBEXECDIR`
-  if test -z `$APXS -q SYSCONFDIR`; then
-    INSTALL_IT="\$(mkinstalldirs) '$APXS_LIBEXECDIR' && \
-                 $APXS -S LIBEXECDIR='$APXS_LIBEXECDIR' \
-                       -i -n php"
-  else
-    APXS_SYSCONFDIR='$(INSTALL_ROOT)'`$APXS -q SYSCONFDIR`
-    INSTALL_IT="\$(mkinstalldirs) '$APXS_LIBEXECDIR' && \
-                \$(mkinstalldirs) '$APXS_SYSCONFDIR' && \
-                 $APXS -S LIBEXECDIR='$APXS_LIBEXECDIR' \
-                       -S SYSCONFDIR='$APXS_SYSCONFDIR' \
-                       -i -a -n php"
-  fi
+
+  INSTALL_IT="\$(mkinstalldirs) '$APXS_LIBEXECDIR' && \
+               $APXS -S LIBEXECDIR='$APXS_LIBEXECDIR' \
+                     -i -n php8"
 
   LIBPHP_CFLAGS="-shared"
   PHP_SUBST(LIBPHP_CFLAGS)
